internal/controller/cache/cache.go

```diff
@@ -0,0 +1,44 @@
+package cache
```

Comment:
is there any reason for not placing this package in `/pkg` directory?

Reply:
I think cache is more or less just a configuration of existing packages so it would make sense to keep it internal. what do you think?